.editorconfig

```diff
@@ -0,0 +1,14 @@
+# Editor configuration, see http://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
This will conflict with go formatting. Do we need to commit this file?

Reply:
Better we keep. 

I updated indentation for go

```
[*.go]
indent_size = 4
```